lib/smart_answer_partial_template_wrapper.rb

```diff
@@ -0,0 +1,65 @@
+module SmartAnswerPartialTemplateWrapper
+  include ActionView::Helpers::TagHelper
+
+  def render(context, options, block)
+    content = super(context, options, block)
+    return content unless valid_path_and_format?(context)
+
+    content_tag(
+      :div,
+      govspeak_to_html(content),
+      data: { debug_template_path: smart_answer_partial_path }
+    )
+  end
+
+private
+
+  def valid_path_and_format?(context)
+    smart_answer_partial_path? && html_format?(context)
+  end
+
+  def html_format?(context)
+    context.request&.format&.symbol.to_s == "html"
```

Comment:
I think that the use of `&.` is a bit of a code "smell". I would've though that `context.request.format` would always return something and I'm not convinced it necessary to convert the format into a symbol and then into into a String in order to do the comparison, e.g. could we compare it to an instance of a `Mime::Type`?

Reply:
This should have been removed some other options were experimented with.. and this led to this..  Once the controller is supplied the format will exist.